tests/JsonSchema/Command/JsonSchemaGenerateCommandTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Tests\JsonSchema\Command;
+
+use ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity\Dummy;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\Console\Tester\ApplicationTester;
+
+/**
+ * @author Jacques Lefebvre <jacques@les-tilleuls.coop>
+ */
+class JsonSchemaGenerateCommandTest extends KernelTestCase
+{
+    /**
+     * @var ApplicationTester
+     */
+    private $tester;
+
+    protected function setUp(): void
+    {
+        self::bootKernel();
+
+        $application = new Application(static::$kernel);
+        $application->setCatchExceptions(true);
+        $application->setAutoExit(false);
+
+        $this->tester = new ApplicationTester($application);
+    }
+
+    public function testExecuteWithoutOption()
+    {
+        $this->tester->run(['command' => 'api:json-schema:generate', 'resource' => Dummy::class]);
+
+        $this->assertJson($this->tester->getDisplay());
+    }
+
+    public function testExecuteWithItemOperationGet()
+    {
+        $this->tester->run(['command' => 'api:json-schema:generate', 'resource' => Dummy::class, '--itemOperation' => 'get']);
```

Comment:
If case we're running the MongoDB build, you must test with the Dummy document instead of the Dummy entity.

ping @alanpoulain, we need to do something about this, it annoys almost all contributors, and is very difficult to debug.